templates/publishing/src/blog-app.html

```diff
@@ -40,6 +40,7 @@
 
       :host {
         --app-primary-color: #404040;
+        -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
```

Comment:
Then we can remove the `-webkit-tap-highlight-color: transparent` on `paper-elements`?